shell/common/shell_test_platform_view.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/common/shell_test_platform_view.h"
+
+#if OS_FUCHSIA
```

Comment:
I'd suggest adding a layer of abstraction over the platform and the client rendering API. In this instance, there is no reason Vulkan needs to be tied to Fuchsia. In fact, @iskakaushik expressed interest in setting up the test harness to use Vulkan on the host (using SwiftShader just like we test OpenGL in a cross platform manner) . This will allow us to test Vulkan functionality on the host in a cross platform manner with Fuchsia being just one of the users of Vulkan.